soccer/src/soccer/planning/primitives/replanner.cpp

```diff
@@ -57,6 +60,25 @@ Trajectory Replanner::full_replan(const Replanner::PlanParams& params) {
         CreatePath::rrt(params.start.linear_motion(), params.goal, params.constraints.mot,
                         params.start.stamp, params.static_obstacles, params.dynamic_obstacles);
 
+    // if the initial path is empty, the goal must be blocked
+    // try to shift the goal_point until it is no longer blocked
+    int max_tries = 10;  // try max this many times before giving up (and sending NOP)
+    LinearMotionInstant almost_goal = params.goal;
+    rj_geometry::Point shift_dir = (params.start.position() - params.goal.position).normalized();
+    double shift_size = 2.0 * kRobotRadius;
```

Comment:
```suggestion
    double shift_size = 4.0 * kRobotRadius;
```
One change I played around with which slightly helped reduce number of collisions

Reply:
This is interesting. Two things: 
1) Can you change the start point (the definition of almost_goal on line 66) of the iteration to be `params.goal + 4.0 * kRobotRadius * shift_dir` but keep the shift_size at 2.0 * kRobotRadius? I think this will have the same effect but keep a higher resolution.
2) Did you run [autoref](https://github.com/robotics-erlangen/autoref) to count collision fouls or just count by hand? If you haven't run the autoref with our sim before, the installation is similar to er-force's simulator, and it will launch a GUI. Make sure you configure the vision port to 10020 because that's what the simulator publishes to. You shouldn't have to connect game controller to see the crash fouls.